src/content/nft/index.md

```diff
@@ -0,0 +1,387 @@
+
+To clarify a few things:
+
+- By pressing the button to create or buy an NFT you are not directly releasing carbon into the atmosphere.
+- NFTs aren't directly increasing the carbon footprint of Ethereum.
+
+Here's where things are going to get a little technical, but bear with us...
+
+### Don't blame it on the NFTs
+
+The whole NFT ecosystem works because Ethereum is decentralized and tamper-proof.
+
+Decentralized meaning you and everyone else can verify you own something without trusting or granting custody to a centralized third party who can impose their own rules at will. It also means your NFT is portable across many different products and markets.
+
+Tamper-proof meaning no one can copy/paste your NFT or steal it.
+
+These qualities of Ethereum makes digitally owning unique items and monetising your content fairly possible.
+
+Blockchains like Bitcoin and Ethereum are energy intensive right now because it takes a lot of energy to preserve these qualities. If it was easy to rewrite Ethereum's history to steal NFTs or cryptocurrency, the system collapses.
+
+<!-- [Need: how do centralized systems do this and why is it worse?]
+ -->
+
+#### The work in minting an NFT
+
+When you mint an NFT, a few things have to happen:
+
+- it needs to be confirmed as an asset on the blockchain.
+- the owner's account balance must be updated to include that asset. This makes it possible for it to then be traded or verifiably "owned".
+- the transactions that confirm the above need to be added to a block and "immortalised" on the chain.
+- the block needs to be confirmed by everyone in the network as "correct". This consensus is what makes it possible for NFT creators to maximise their earnings – there are no intermediaries needed.
+
+These tasks are done by miners. And they let the rest of the network know about your NFT and who owns it. This means mining needs to be sufficiently difficult, otherwise anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership. There are lots of incentives in place to make sure miners are acting honestly.
+
+[More on mining](/developers/docs/consensus-mechanisms/pow/)
+
+#### Making the NFT tamper-proof
+
+Mining difficulty comes from the fact that it takes a lot of computing power to create new blocks in the chain. Importantly, blocks are created consistently, not just when they're needed. They're created every 12 seconds or so.
+
+This is important for making Ethereum tamper-proof, one of the qualities that makes NFTs possible. The more blocks the more secure the chain. If your NFT was created in block #600 and a hacker were to try and change themselves to the creator of your NFT, it would immediately change the digital fingerprint of all subsequent blocks, #601, #602, #603, and so on. That means anyone running Ethereum software would immediately be able to tell and prevent it from happening.
```

Comment:
```suggestion
This is important for making Ethereum tamper-proof, one of the qualities that makes NFTs possible. The more blocks the more secure the chain. If your NFT was created in block #600 and a hacker were to try and change themselves to the creator of your NFT, it would immediately change the digital fingerprint of all subsequent blocks, #601, #602, #603, and so on. That means anyone running Ethereum software would immediately be able to detect and prevent it from happening.
```